components/ee/ws-scheduler/pkg/scaler/metrics.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 TypeFox GmbH. All rights reserved.
+// Licensed under the Gitpod Enterprise Source Code License,
+// See License.enterprise.txt in the project root folder.
+
+package scaler
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type metrics struct {
+	setpointGauge   prometheus.Gauge
+	ghostCountGauge prometheus.Gauge
+}
+
+func newMetrics() *metrics {
+	return &metrics{
+		setpointGauge: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "driver_setpoint",
+			Help: "setpoint of the scaler driver",
+		}),
+		ghostCountGauge: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "ghost_count",
```

Comment:
Nit:

the `_count` suffix is used [by histograms](https://prometheus.io/docs/concepts/metric_types/#histogram). Maybe it's because I'm just used to associate them, but it feels strange to see a gauge with that suffix.

On the other hand, gauges can also [represent counters that go up and down](https://prometheus.io/docs/concepts/metric_types/#gauge) and I can't think of any good suffix to replace that. So feel free to just ignore this comment, to replace the suffix with something else, or just remove the suffix if you prefer :slightly_smiling_face:

Reply:
💯 

I had similar thoughts when choosing `_count`. I thought for a moment to use `_total`, but that seems reserved for counter.

Reply:
Just took a look at [OpenMetrics](https://github.com/OpenObservability/OpenMetrics) and searched for the standard about naming and [suffixes](https://github.com/OpenObservability/OpenMetrics/blob/master/specification/OpenMetrics.md#suffixes).

For a gauge, there are no suffixes. Usually, they just end with the unit. E.g.: `node_memory_MemFree_bytes`. Since our gauge doesn't have a unit, maybe we can just rename it to `gitpod_ws_scaler_ghost_workspaces`